src/lib/services/fileChange/fileChangeHandlers/typeScriptFileChangeHandler.ts

```diff
@@ -8,13 +8,16 @@ import ModuleContext from '../../../interfaces/moduleContext';
 import I18nextScannerModule from '../../../modules/i18nextScanner/i18nextScannerModule';
 import ModuleChainManager from '../../../modules/moduleChainManager';
 import CodeTranslationStore from '../../../stores/codeTranslation/codeTranslationStore';
+import FileLocationStore from '../../../stores/fileLocation/fileLocationStore';
 
-export default class TypeScriptFileChangeHandler implements FileChangeHandler {
+export default class TypeScriptFileChangeHandler extends FileChangeHandler {
```

Comment:
This entire file should be renamed. There should be no reference to typescript. It should just be a CodeFileChangeHandler.